packages/@vue/cli/bin/vue.js

```diff
@@ -261,7 +258,14 @@ if (!process.argv.slice(2).length) {
 function suggestCommands (unknownCommand) {
   const availableCommands = program.commands.map(cmd => cmd._name)
 
-  const suggestion = didYouMean(unknownCommand, availableCommands)
+  let suggestion
+
+  availableCommands.forEach(cmd => {
+    if (leven(cmd, unknownCommand) < 3) {
+      suggestion = cmd
```

Comment:
You still didn't compare two possible suggestions.

Reply:
It would be great if you could give me a better picture regarding the implementation strategy that you have in mind.

Reply:
```js
  availableCommands.forEach(cmd => {
    if (leven(cmd, unknownCommand) < 2) {
      //
   }
})
```
This would find the closest match. To get going ahead can you suggest a possible implementation?

Reply:
"Closest" doesn't mean less than a certain threshold. It means the distance is the smallest. So I have suggested to "compare" and "sort" the distances.